### PR TITLE
Add W3C's Jigsaw

### PR DESCRIPTION
**Pull request by Retia-Adolf**

### CODE REVIEW

File: jigsaw.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        ],
+        [
+            "scripts\\jigadmin.bat",
+            "Jigsaw\\Start JigAdmin Client"
+        ]
+    ],
+    "persist": [
+        "Jigsaw"
+    ],
+    "checkver": "<!-- news section-->\\s+<!-- Jigsaw release -->\\s+<ul>\\s+<li><b>Released \\w{3,9} \\d+\\w+, \\d{4}: Jigsaw (?<version>((\\d+\\.)+\\d+))",
```
Comment (rasa) [inline jigsaw.json]: This could be simplified

File: jigsaw.json [hunk 2]
```diff
@@ -31,7 +31,7 @@
     "persist": [
         "Jigsaw"
     ],
-    "checkver": "<!-- news section-->\\s+<!-- Jigsaw release -->\\s+<ul>\\s+<li><b>Released \\w{3,9} \\d+\\w+, \\d{4}: Jigsaw (?<version>((\\d+\\.)+\\d+))",
+    "checkver": "<!-- news section-->\\s+<!-- Jigsaw release -->\\s+<ul>\\s+<li><b>[ ,:\\w\\d]+(?<version>((\\d+\\.)+\\d+))",
```
Comment (r15ch13) [inline jigsaw.json]: Keep it simple and use [`Released[^:]+:\s+Jigsaw\s+([\d.]+)`](https://regex101.com/r/dKrZkh/1) or even simpler `jigsaw_([\d.]+).zip`

File: jigsaw.json [hunk 3]
```diff
@@ -31,12 +31,12 @@
...
     ],
-    "checkver": "<!-- news section-->\\s+<!-- Jigsaw release -->\\s+<ul>\\s+<li><b>[ ,:\\w\\d]+(?<version>((\\d+\\.)+\\d+))",
+    "checkver": "jigsaw_[\\w.]+\\.zip",
     "autoupdate": {
-        "url": "https://jigsaw.w3.org/Distrib/jigsaw_$matchVersion.zip",
+        "url": "https://jigsaw.w3.org/Distrib/jigsaw_$version.zip",
         "hash": {
             "url": "$url",
-            "find": "(?:<dt>$basename</dt>\\s+<dd>.*</dd>\\s+<dd>md5sum:\\s*)([a-fA-F0-9]{32})(?:\\s*</dd>\\s+<dt>jigsaw_$matchVersion+tar\\.gz)"
+            "find": "md5sum:\\s*([a-F\\d]{32})"
```
Comment (r15ch13) [inline jigsaw.json]: `md5sum:\\s+([a-fA-F\\d]{32})`

File: jigsaw.json [hunk 4]
```diff
@@ -36,7 +36,7 @@
         "url": "https://jigsaw.w3.org/Distrib/jigsaw_$version.zip",
         "hash": {
             "url": "$url",
-            "find": "md5sum:\\s*([a-F\\d]{32})"
+            "find": "md5sum:\\s+([a-F\\d]{32})"
```
Comment (r15ch13) [inline jigsaw.json]: `a-F` to `a-fA-F` (also, it would match xyz but not XYZ)

File: jigsaw.json [hunk 5]
```diff
@@ -36,7 +36,7 @@
         "url": "https://jigsaw.w3.org/Distrib/jigsaw_$version.zip",
         "hash": {
             "url": "$url",
-            "find": "md5sum:\\s+([a-F\\d]{32})"
+            "find": "md5sum:\\s+([A-f\\d]{32})"
```
Comment (r15ch13) [inline jigsaw.json]: Also matches XYZ, please use `a-fA-F`

**Pull request merged**